cMake/FindPySideTools.cmake

```diff
@@ -20,7 +20,7 @@ if(WIN32 OR ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     set(PYSIDE_BIN_DIR ${PYTHON_BIN_DIR})
 endif(WIN32 OR ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 
-FIND_PROGRAM(PYSIDEUIC4BINARY NAMES pyside-uic pyside-uic-${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR} HINTS ${PYSIDE_BIN_DIR})
+FIND_PROGRAM(PYSIDEUIC4BINARY NAMES python2-pyside-uic pyside-uic pyside-uic-${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR} HINTS ${PYSIDE_BIN_DIR})

```

Comment:
For Arch witch I am familiar with "python" == python v3. python2 is used to for legasy code along side python v3. When switching to Python3 the naming would likely either be python-pyside-uic or pyside-uic. Taking this into consideration i think that python${PYTHON_VERSION_MAJOR} will not work on Arch Linux.